.github/dependabot.yml

```diff
@@ -7,10 +7,23 @@ updates:
     day: "monday"
     time: "09:00"
     timezone: "Europe/Berlin"
+  commit-message:
+    prefix: "ci"
+  labels:
+    - run-ci
```

Comment:
Are we sure that we want to always run those PRs automatically? I thought we had some discussions about this in the past and decided against it.